src/colorizer/ColorizeCanvas.ts

```diff
@@ -225,15 +226,16 @@ export default class ColorizeCanvas implements IRenderCanvas {
 
     this.canvasResolution = new Vector2(1, 1);
     this.categoricalPalette = new ColorRamp(["black"]);
-    this.currentFrame = 0;
+    this.currentFrame = -1;
```

Comment:
I like the change from 0(frame 0 loaded) to -1(no frame loaded) here! Creates a clearer distinction.